tool/src/main/java/org/wildfly/security/tool/ElytronToolMessages.java

```diff
@@ -607,4 +607,9 @@ public interface ElytronToolMessages extends BasicLogger {
     @Message(id = 35, value = "Only one of '%s' and '%s' can be specified at the same time")
     IllegalArgumentException mutuallyExclusiveOptions(String first, String second);
 
+    @Message(id = NONE, value = "To get list of options for specific command, please specify the command by using ./elytron-tool.sh [command] --help")
```

Comment:
Just a minor, this should be placed before the `Numeric Errors`  comment

Reply:
Done

Reply:
s/get list of options for specific command/get a list of options for a specific command